CSS/3-recent-works.css

```diff
@@ -2,10 +2,13 @@
 
 .recent_works_container {
```

Comment:
The margins above and below the first project card don't quite match those from the template design.

Here's your design:

<img width="723" alt="Screen Shot 2022-06-01 at 13 12 10" src="https://user-images.githubusercontent.com/82426682/171473764-9a42e5bc-8616-4c32-94a5-c6f399af5224.png">

Here's the template design:

<img width="707" alt="Screen Shot 2022-06-01 at 13 12 45" src="https://user-images.githubusercontent.com/82426682/171473858-922bdc34-d243-4113-9ce0-d1191db7f4d9.png">

Do you see how different the margins are? Kindly fix this.